src/hypothesis/core.py

```diff
@@ -443,6 +449,15 @@ def __init__(self, test_runner, search_strategy, test, settings, random):
         self.random = random
 
     def evaluate_test_data(self, data):
+        if (
+            time.time() - self.start_time >= HUNG_TEST_TIME_LIMIT
+        ):
+            fail_health_check(self.settings, (
+                'Your test has been running for at least five minutes. This '
+                'is probably not what you intended, so by default Hypothesis '
+                'turns it into an error.'
```

Comment:
Does the `fail_health_check` function include instructions for how to disable the health check?